rtgui/lensgeom.cc

```diff
@@ -113,3 +113,19 @@ void LensGeometry::setBatchMode (bool batchMode)
     ToolPanel::setBatchMode (batchMode);
     removeIfThere (this, autoCrop);
 }
+
+void LensGeometry::disableAutoFillIfActive ()
+{
+
+    if (!batchMode) {
+        if (fill->get_active()) {
+            fillConn.block (true);
```

Comment:
Trying to prevent the mutex issue, I changed this part to:
```
            disableListener();
            fillConn.block (true);
            fill->set_active(false);
            fillConn.block (false);
            enableListener();
            if (listener) {
                listener->panelChanged (EvTransAutoFill, M("GENERAL_DISABLED"));
            }
```
But the mutex problem is still there.